CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the like and repost counts from the Main and Profile feeds.
 - Fixed an issue where the sheet asking users to set up a NIP-05 username would appear after reinstalling Nos, even if the profile already had a NIP-05 username.
 - Fixed a bug where urls with periods after them would include the period.
-- Replaced hard-coded color values.
+- Replaced hard-coded hex and white: opacity: color values.
```

Comment:
I think the original is better -- we don't need so much detail in the CHANGELOG.
```suggestion
- Replaced hard-coded color values.
```

Reply:
Understood, I will change this in the final PR for RGB values.

Reply:
Let's change it now and skip the changelog update in the future PRs.